sql/core/src/test/scala/org/apache/spark/sql/execution/command/DDLSuite.scala

```diff
@@ -2078,23 +2078,6 @@ abstract class DDLSuite extends QueryTest with SQLTestUtils {
     }
   }
 
-  test("show columns - negative test") {
```

Comment:
This test now has nos sense, because it assumes that you can put a multipart table identifier with a namespace defined